pyinstaller-gui.py

```diff
@@ -76,7 +76,11 @@ def killapp(self, event):
         sys.exit(0)
 
     def makePackage(self, event):
-        commands = ['python3', 'pyinstaller.py']
+        if sys.platform.startswith('win'):
+            pythoncmd = ['py', '-3']
```

Comment:
> So if 3.0-3.2 have to be supported, another way to acces python3 is needed.
> Maybe directly from the registry?

@Yegers I think we are going to support python 3.3+.

But what about getting python command from sys.executable?